compiler/dalgona/CMakeLists.txt

```diff
@@ -1,10 +1,33 @@
+find_package(PythonInterp 3.8 QUIET)
+find_package(PythonLibs 3.8 QUIET)
+
+if(NOT ${PYTHONINTERP_FOUND})
+  message(STATUS "Build dalgona: FAILED (Python3 is missing)")
+  return()
+endif()
+
+if(${PYTHON_VERSION_MINOR} LESS 8)
```

Comment:
Our development platform starts from Ubuntu 18.04 which has python 3.6 as default.
This requires to install 3.8 as default or somehow use it with alternatives.
Can we start from 3.6?

Reply:
The main reason I used 3.8 is to match with our `venv` environment, which is used for tests.

I guess that lowering the bar into 3.6 would be ok. I'll test it in the full draft PR to make sure it works in the CI worker.

Reply:
> The main reason I used 3.8 is to match with our venv environment, which is used for tests.

The viewpoint of mine is that the `venv` is execution environment and this check is of build environment. Can build with 3.6 and run with 3.8 be possible and valid?

Reply:
> Can build with 3.6 and run with 3.8 be possible and valid?

I've locally tested it, but it was impossible. Python library (ex: libpython3.6m.so.1.0 or libpython3.8.so.1.0) linked during the build time should match with execution environment.

`common-artifacts` already requires python 3.8 (even for Ubuntu 18.04 environment), so I think it would be better to use 3.8 in `dalgona` too.